Documentation/gettingstarted/k8s-install-gke.rst

```diff
@@ -101,6 +101,4 @@ to the cluster. The NodeInit DaemonSet will perform the following actions:
 
 .. include:: k8s-install-restart-pods.rst
 .. include:: k8s-install-gke-validate.rst
-.. include:: namespace-kube-system.rst
-.. include:: hubble-enable.rst
-
+.. include:: next-steps.rst
```

Comment:
Are the `k8s-install.gke.rst` and friends still referenced in the documentation? I could only find references to them from `install/kubernetes/cilium/README.*` and `.github/ISSUE_TEMPLATE/release_template.md` (which I think we should update to the new guides as well).

I suppose we can remove them if they are not?